src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -46,21 +58,19 @@
 #define CHIP_CLUSTER_NETWORK_COMMISSIONING_MAX_NETWORKS 4
 #endif // CHIP_CLUSTER_NETWORK_COMMISSIONING_MAX_NETWORKS
 
-using namespace chip;
-using namespace chip::app;
-using namespace chip::app::clusters;
-using namespace chip::app::clusters::NetworkCommissioning;
+using namespace ::chip;
+using namespace ::chip::app;
+using namespace ::chip::app::clusters;
+using namespace ::chip::app::clusters::NetworkCommissioning;
+using namespace ::chip::app::clusters::NetworkCommissioning::Internal;
 
 namespace chip {
 namespace app {
 namespace clusters {
 namespace NetworkCommissioning {
 
-constexpr uint8_t kMaxNetworkIDLen       = 32;
-constexpr uint8_t kMaxThreadDatasetLen   = 254; // As defined in Thread spec.
-constexpr uint8_t kMaxWiFiSSIDLen        = 32;
-constexpr uint8_t kMaxWiFiCredentialsLen = 64;
-constexpr uint8_t kMaxNetworks           = CHIP_CLUSTER_NETWORK_COMMISSIONING_MAX_NETWORKS;
+// Internal structs and helper functions for network commissioning.
+namespace Internal {
```

Comment:
Can this be an unnamed namespace?

Reply:
This should definitely be an anonymous namespace.